src/components/UiPhoto/index.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <div class="ui-photo">
+    <div class="ui-photo-clipper">
+      <ui-photo-field
+        v-if="!isEditorOpened"
+        name="ui-photo-clipper-img"
+        :value="originImg"
+        :note="note"
+        :document-type="documentType"
+        :label="label"
+        :min-width="minWidth"
+        :min-height="minHeight"
+        :disabled="disabled"
+        :file-extensions="IMAGE_FILE_EXTENSIONS"
+        :max-size="maxSize"
+        :error-message="errorMessage"
+        @input="tryCropImg"
+      />
+
+      <template v-if="isEditorOpened">
+        <div
+          class="ui-photo-clipper__label"
+          :class="{
+            'ui-photo-clipper__label--disabled': $attrs.disabled,
+            'ui-photo-clipper__label--error': errorMessage
+          }"
+        >
+          {{ labelMain }}
+        </div>
+
+        <div class="ui-photo-clipper__editor">
+          <clipper-basic
+            class="ui-photo-clipper__clipper"
+            :src="imgURL"
+            ref="clipper"
+            bg-color="transparent"
+            :ratio="ratio"
+          >
+            <div
+              class="ui-photo-clipper__no-image"
+              slot="placeholder">
+              <!-- eslint-disable max-len -->
+              <i class="mdi mdi-face-recognition ui-photo-clipper__icon ui-photo-clipper__reset-icon" />
+              <!-- eslint-enable max-len -->
+            </div>
+          </clipper-basic>
+
+          <div class="ui-photo-clipper__actions">
+            <button
+              class="app__button-flat"
+              type="button"
+              v-if="!$attrs.disabled"
+              :title="'clipper-field.save-lbl'"
+              :disabled="disabled"
+              @click="cropImg"
+            >
+              {{ labelSave }}
+            </button>
+            <button
+              class="app__button-flat ui-photo-clipper__reset-btn"
+              type="button"
+              v-if="!$attrs.disabled"
+              :title="'clipper-field.cancel-lbl'"
+              :disabled="disabled"
+              @click="reset"
+            >
+              {{ labelCancel }}
+            </button>
+          </div>
+        </div>
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+import FileField from './FileField'
+import { clipperBasic } from 'vuejs-clipper'
+import Vue from 'vue'
+import VueRx from 'vue-rx'
+import '@mdi/font/css/materialdesignicons.css'
+import { Document } from '@tokend/js-sdk'
+
+Vue.use(VueRx)
+
+const MAX_FILE_MEGABYTES = 32
+const IMAGE_FILE_EXTENSIONS = ['jpg', 'png', 'jpeg']
+
+export default {
+  components: {
+    'ui-photo-field': FileField,
```

Comment:
don't register components under different names. If it's called `FileField`, it should remain `<file-field>` in your template